grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -247,19 +267,19 @@ export class ClientPageStore extends ComponentStore<ClientPageState> {
           );
 
   /** An effect fetching a client with a given id and updating the state. */
-  readonly selectClient = this.effect<string>(
+  readonly fetchClient = this.effect<void>(
```

Comment:
Please change the visiblity to `private` to hide implementation details.

Reply:
Of course!